api/src/main/java/me/lokka30/treasury/api/economy/account/accessor/PlayerAccountAccessor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/lokka30/Treasury>.
+ */
+
+package me.lokka30.treasury.api.economy.account.accessor;
+
+import java.util.Objects;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import me.lokka30.treasury.api.common.response.Response;
+import me.lokka30.treasury.api.economy.account.PlayerAccount;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents an account accessor, which accesses {@link PlayerAccount player accounts}
+ *
+ * @author MrIvanPlays
+ * @since 2.0.0
+ */
+public abstract class PlayerAccountAccessor {
+
+    private UUID uniqueId;
+
+    protected PlayerAccountAccessor() {
+    }
+
+    /**
+     * Specifies the {@link UUID unique id} of the account owner
```

Comment:
```suggestion
     * Specifies the {@link UUID} of the account owner
```

Reply:
no

Reply:
> no

IMO it's an unnecessary elaboration. All of the IDs are unique anyways.

Reply:
ik but it looks better in the docs

Specifies the [unique id](https://mrivanplays.com/) vs
Specified the [UUID](https://mrivanplays.com)